launchers/ids-connector/vault.properties

```diff
@@ -0,0 +1,2 @@
+consumer-blob-storage-key=r+lqbmhzLjLysUN/BnV3ZhOr0UXz1FT1MdWSCoXHIvUXrCp6zH9CYS5cfMlFoxRVwfXoogXNP3j4u4P7ZMQp3Q==
```

Comment:
You've just leaked your blob store key. NEVER check in sensitive information!!!

Reply:
Yea, totally forgot about this :)